### PR TITLE
Install dep (github.com/golang/dep)

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: Dockerfile-1.7 [hunk 1]
```diff
@@ -26,19 +26,15 @@ RUN mkdir /release && \
     chown $USER /release
 
 # create directory to get source from
-RUN mkdir /src && \
-    chown $USER /src && \
-    mkdir -p /go/src/github.com/decred/dcrd && \
```
Comment (jcvernaleo) [inline Dockerfile-1.7]: Why did you remove these?  The `run_tests.sh` script requires them (was done that way to make it work on travis more seemlessly).

File: Dockerfile-1.8 [hunk 2]
```diff
@@ -26,19 +26,15 @@ RUN mkdir /release && \
     chown $USER /release
 
 # create directory to get source from
-RUN mkdir /src && \
-    chown $USER /src && \
-    mkdir -p /go/src/github.com/decred/dcrd && \
```
Comment (jcvernaleo) [inline Dockerfile-1.8]: Same as for the 1.7

Comment (dajohi) [inline Dockerfile-1.8]: fails to docker build for me.  permission denied.

Comment (dajohi) [inline Dockerfile-1.8]: OK, have a look now...
